WordPress/twentytwentyfive
# Add pattern: Contact: Info and locations

**Description**
Adds the Contact - Info and locations pattern.
[Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1814&t=CjlGc0FElSt2S1sr-0)
Closes https://github.com/WordPress/twentytwentyfive/issues/61


**Testing Instructions**
Add the pattern and check if it matches the design.
Also change the width of the browser window and check that it still looks correct.
